### PR TITLE
Update reaction content type

### PR DESCRIPTION
**Pull request by rygine**
Just updating the content type with this PR and adding the now required `schema` to the reaction message. In a follow up, we'll add support for `unicode` and `custom` reactions.

### CODE REVIEW

File: src/model/reactions.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export async function addReaction(
     action: "added",
     reference: message.xmtpID,
     content: reactionName,
+    schema: "unicode",
```
Comment (nakajima) [inline src/model/reactions.ts]: Aren't we currently using shortcodes?

Comment (rygine) [reply src/model/reactions.ts]: i should read the code better 🙃

Comment (rygine) [reply src/model/reactions.ts]: [fixed](https://github.com/xmtp/xmtp-react-playground/pull/7/commits/b046eb86adcdd9519a1448ebc17fe7f3bc7224d2)

**Comment by nplasterer** [comment]
Can you add a fallback text in this PR for reactions as well 🙏

**Review by nplasterer — APPROVED** [review]

**Pull request merged**